src/OpenLoco/Vehicles/VehicleBogie.cpp

```diff
@@ -1,16 +1,145 @@
+#include "../Entities/EntityManager.h"
 #include "../Interop/Interop.hpp"
 #include "Vehicle.h"
 
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Vehicles
 {
+    static loco_global<VehicleBogie*, 0x01136124> vehicleUpdate_frontBogie;
+    static loco_global<VehicleBogie*, 0x01136128> vehicleUpdate_backBogie;
+    static loco_global<bool, 0x01136237> vehicleUpdate_frontBogieHasMoved; // remainingDistance related?
+    static loco_global<bool, 0x01136238> vehicleUpdate_backBogieHasMoved;  // remainingDistance related?
+    static loco_global<int32_t, 0x0113612C> vehicleUpdate_var_113612C;     // Speed
+    static loco_global<uint32_t, 0x01136114> vehicleUpdate_var_1136114;
+    static loco_global<int32_t, 0x01136130> vehicleUpdate_var_1136130; // Speed
+    static loco_global<EntityId, 0x0113610E> vehicleUpdate_collisionCarComponent;
+
     // 0x004AA008
     bool VehicleBogie::update()
+    {
+        vehicleUpdate_backBogie = vehicleUpdate_frontBogie;
```

Comment:
I know this is probably how the asm reads, but would there be any reason why these would need to be set if they aren't used for air/water, since those modes just return true below?

Reply:
Yeah these globals are used throughout the update code you would break a whole bunch of things.